cmd/rook/block/block_test.go

```diff
@@ -25,14 +25,26 @@ import (
 func TestBlockCommand(t *testing.T) {
 	assert.Equal(t, "block", Cmd.Use)
 
-	commands := []string{}
+	var expectedCommands = []string{}
+	if runtime.GOOS == "linux" {
+		expectedCommands = []string{"create", "delete", "ls", "map", "unmap"}
+	} else {
+		expectedCommands = []string{"create", "ls"}
+	}
+
+	commandTested := 0
 	for _, command := range Cmd.Commands() {
-		commands = append(commands, command.Name())
+		assert.Equal(t, true, stringInSlice(command.Name(), expectedCommands))
```

Comment:
a perhaps more simple way to do this would be to sort the `cmd.Commands()` slice and compare the `expectedCommands` against that.